.github/workflows/test_action.yml

```diff
@@ -0,0 +1,25 @@
+# This is a basic test workflow to trigger an echo action on a push event
+
+name: Echo_Event
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push event but only for the "development" branch
+  push:
+    branches: [ "development" ]
```

Comment:
this is a really nice start! 
Now it will run the workflow whenever anything is merged into the development branch.

you can also add
```
pull_request:
    branches: [ development ]
```
try that, and you will see the action run in any branch that is a pull-request to the development branch (i.e. this branch)
So keep your PR open, but just add the new changes to it, and you'll see the action run.

Reply:
Yes pls, i will try that.